src/updater.cpp

```diff
@@ -970,7 +973,7 @@ namespace appimage {
                     trim(line, '\n');
                     trim(line);
 
-                    d->issueStatusMessage(std::string("gpg2: ") + line);
+                    d->issueStatusMessage(std::string("gpg: ") + line);
```

Comment:
You should reuse `gpgPath` here. This'll allow users to see which binary was actually used.

Reply:
ah,  good idea. :smile: